strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationControllerTestIT.java

```diff
@@ -667,4 +696,96 @@ public void whenStorageIsCreatedWithoutBasedirProvidedDefaultIsSet(String extens
+                                                                 repositoryId = NUGET_REPOSITORY_RELEASES)
+                                                Repository repository,
+                                                @NugetTestArtifact(storageId = NUGET_STORAGE_ID,
+                                                                   repositoryId = NUGET_REPOSITORY_RELEASES,
+                                                                   id = "Org.Carlspring.Strongbox.Examples.Nuget.Mono.Delete",
+                                                                   versions = "1.0.0")
+                                                Path path)
+            throws Exception
+    {
+        deleteRepositoryPath(repository, path);
+    }
+
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
+    @Test
+    public void testDeleteRawPathInRepository(@TestRepository(layout = RawLayoutProvider.ALIAS,
+                                                              repositoryId = RAW_REPOSITORY_RELEASES)
+                                              Repository repository,
+                                              @TestArtifact(repositoryId = RAW_REPOSITORY_RELEASES,
+                                                            resource = "org/foo/bar/blah.zip",
+                                                            generator = NullArtifactGenerator.class)
+                                              Path path)
+            throws Exception
+    {
+        deleteRepositoryPath(repository, path);
+    }
+
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class,
+                  ArtifactManagementTestExecutionListener.class })
+    @Test
+    public void testDeleteNpmPathInRepository(@NpmRepository(repositoryId = NPM_REPOSITORY_RELEASES)
+                                                             Repository repository,
+                                              @NpmTestArtifact(repositoryId = NPM_REPOSITORY_RELEASES,
+                                                               versions = "1.0.0",
+                                                               id = "npm-test-release",
+                                                               scope = "@carlspring")
+                                              Path path)
+            throws Exception
+    {
+        deleteRepositoryPath(repository, path);
+    }
+
+    private void deleteRepositoryPath(Repository repository, Path path)
+            throws IOException
+    {
+        String url = getContextBaseUrl() + "/delete";
+
+        RepositoryPath repositoryPath = (RepositoryPath) path.toRealPath();
+        String repositoryPathStr = RepositoryFiles.relativizePath(repositoryPath);
+
+        PathForm pathForm = buildPathForm(repository.getStorage().getId(), repository.getId(), repositoryPathStr, false);
```

Comment:
Cosmetic: Could you please, chop these parameters as one per line and indent them properly? Thanks! :)